src/js/Edit/L.PM.Edit.Line.js

```diff
@@ -259,7 +260,9 @@ Edit.Line = Edit.extend({
     // first, make this middlemarker a regular marker
     newM.off('movestart');
     newM.off('click');
-
+    newM.on('click', () => {
+      newM.on('click', this._onVertexClick, this);
+    })
```

Comment:
This makes no sense for me. MiddleMarkers are also created over `_createMarker` so it has already the click event for `_onVertexClick`. Also it don't understand why the marker have to clicked twice?

Reply:
Thats what I tought aswell, but if I create a marker from middlemarker it does not have the event.

Reply:
And 2 click events was my bad. fixed